pkg/controller/clustersecrets/reconcilerequest.go

```diff
@@ -80,15 +80,15 @@ func (r *reconcileRequest) secretList(ctx context.Context) (*corev1.SecretList,
 	labelSelector := r.instance.Spec.Source.LabelSelector
 	fieldSelector := r.instance.Spec.Source.FieldSelector
 
-	opts := client.InNamespace(r.instance.Spec.Source.Namespace)
+	opts := &client.ListOptions{Namespace: r.instance.Spec.Source.Namespace}
```

Comment:
why did this have to change?

Reply:
`InNamespace` isn't a function any more. And the signature for List() function parameters changed https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#controller-runtime-api-updates